task_02/danil_dyachenko/02_knight_and_princess/graph_gen.cpp

```diff
@@ -0,0 +1,107 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+
+struct Vertex{
+    int id;
```

Comment:
Для лучшей выразительности языка можно `int` заменить на кастомный тип, например
```cpp
using VertexId = int;
```
В таком случае, если понадобится заменить `id` например на `std::string`, это нужно будет сделать только в одном месте.
```suggestion
    VertexId id;
```